service/recognition/noise_reduction_model.py

```diff
@@ -0,0 +1,35 @@
+"""
+Noise-reduction module
+"""
+
+from typing import Any
+
+import torch
+from df.enhance import enhance, init_df
+
+from .model import RecognitionModel
+
+
+class NoiseReductionModel(RecognitionModel):
+    """
+    Class for noise-reduction model
+    """
+    def __init__(self):
+        self.model = self.load_model()[0]
+        self.df_state = self.load_model()[1]
+
+    def load_model(self) -> Any:
+        model, df_state, _ = init_df()
+        return model, df_state
+
+    def recognize(self, audio: torch.Tensor, **kwargs) -> torch.Tensor:
```

Comment:
A solution with a new abstract model will be good, since here the subclass does not match the parent signature of this method